data/json/monsters/insect_spider.json

```diff
@@ -2294,6 +2294,155 @@
+      "reproduction": { "baby_egg": "egg_stag_beetle", "baby_count": 1, "baby_timer": 20 },
+      "flags": [ "DESTROYS", "PUSH_VEH" ]
+    }
+  },
+  {
+    "id": "mon_stag_beetle_giant",
+    "type": "MONSTER",
+    "name": { "str": "massive stag beetle" },
+    "description": "This enormous stag beetle has reached mutated maturity, but continues to grow.  Thankfully, that growth is exposing vulnerable cracks of flesh in the strained chitin.",
+    "default_faction": "stag_beetle",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "400 L",
+    "weight": "600 kg",
+    "diff": 2,
+    "hp": 350,
+    "speed": 80,
+    "material": [ "iflesh" ],
+    "symbol": "S",
+    "color": "brown",
+    "aggression": 40,
+    "morale": 30,
+    "melee_skill": 5,
+    "melee_dice": 6,
+    "melee_dice_sides": 6,
+    "melee_cut": 9,
+    "dodge": 1,
+    "armor_bash": 60,
+    "armor_cut": 72,
+    "armor_bullet": 50,
+    "armor_acid": 18,
+    "armor_fire": 8,
+    "weakpoints": [
+      { "name": "a soft section of chitin", "armor_mult": { "physical": 0.33 }, "coverage": 20 },
+      { "name": "a crack in the chitin", "armor_mult": { "physical": 0 }, "coverage": 5 }
+    ],
+    "mountable_weight_ratio": 6,
+    "vision_day": 14,
+    "vision_night": 12,
+    "harvest": "arachnid",
+    "anger_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "HURT" ],
+    "fear_triggers": [ "FIRE" ],
+    "special_attacks": [
+      {
+        "id": "stag_smash",
+        "throw_strength": 60,
+        "hit_dmg_u": "The %1$s rams you in the %2$s!",
+        "hit_dmg_npc": "The %1$s rams <npcname> in the %2$s!",
+        "no_dmg_msg_u": "The %1$s rams your %2$s, but you hold steady!",
+        "no_dmg_msg_npc": "The %1$s rams <npcname>'s %2$s, but they hold steady!",
+        "miss_msg_u": "The %s flicks it's horns at you, but you dodge!",
+        "miss_msg_npc": "The %s flicks it's horns at <npcname>, but they dodge!"
+      }
+    ],
+    "upgrades": { "age_grow": 28, "into": "mon_stag_beetle_mega" },
+    "reproduction": { "baby_egg": "egg_stag_beetle", "baby_count": 1, "baby_timer": 20 },
+    "baby_flags": [ "SPRING", "SUMMER", "AUTUMN", "WINTER" ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "GOODHEARING",
+      "BASHES",
+      "DESTROYS",
+      "PATH_AVOID_FIRE",
+      "PATH_AVOID_FALL",
+      "PUSH_MON",
+      "PUSH_VEH",
+      "PET_WONT_FOLLOW",
+      "PET_MOUNTABLE"
+    ]
+  },
+  {
+    "id": "mon_stag_beetle_mega",
+    "type": "MONSTER",
+    "name": { "str": "titan stag beetle" },
+    "description": "This stag beetle rivals an armored personnel carrier in size, bearing horns that could fell a tree and chitin thick enough to ignore most bullets.  It's bulky shell looks nigh-impenetrable, and the cracks have sealed with a foul paste.",
+    "copy-from": "mon_stag_beetle_giant",
+    "proportional": { "hp": 4, "speed": 0.75, "morale": 3, "melee_dice_sides": 2 },
+    "volume": "950 L",
+    "weight": "4400 kg",
```

Comment:
```suggestion
    "weight": "1000 kg",
```
Currently, there are some problems with critters weighing over 1000 kg in terms of corpses.

Reply:
I thought those had been resolved?  My testing didn't encounter any issues, and there is the armored centipede weighing 2500 kg.  I was under the impression that the only issue was with volume > 1000 L, which is why I went with 950 (for safety sake).  Admittedly, butchery (of armored centipede and this) does put organs and chitin in 2-4 squares.

Reply:
News to me, issue #44508 is still open

Reply:
Strange, I'll do more testing- nothing has disappeared for me, and the net weight of the corpse always ties with the butchery products (which spread across several tiles) in my testing.  Could someone have accidentally fixed this in a tile overflow fix, by chance?